managers/mongo/src/test/kotlin/com/justai/jaicf/context/manager/test/MongoBotContextManagerTest.kt

```diff
@@ -75,4 +61,40 @@ class MongoBotContextManagerTest {
         assertTrue(result.result is CustomValue)
         assertTrue(result.session["value"] is CustomValue)
     }
-}
\ No newline at end of file
+}
+
+@EnabledIfEnvironmentVariable(named = "mongo", matches = "external")
```

Comment:
I'd suggest use CI-related env variables. Usually `CI: true` env var is used.
And it'd better make `port` env var to, so we'll be able to change CI configuration without changing the code.

So, I'd suggest change this to:
`@EnabledIfEnvironmentVariable("CI", "true")`
`val port = System.getenv("PORT")`